d8/providers/gitlab-pipelines/.gitlab-ci.yml

```diff
@@ -113,7 +121,6 @@ test:visual-regression:
       - backstop_data
   dependencies:
     - configure:env-vars
-    - deploy:pantheon
```

Comment:
Different stages don't really need dependencies so removing this dependency doesn't make any harm